packages/replay-canvas/src/canvas.ts

```diff
@@ -1,14 +1,16 @@
 import { CanvasManager } from '@sentry-internal/rrweb';
 import { convertIntegrationFnToClass } from '@sentry/core';
-import type { CanvasManagerInterface, CanvasManagerOptions } from '@sentry/replay';
+import type { CanvasManagerOptions } from '@sentry/replay';
 import type { IntegrationFn } from '@sentry/types';
 
 interface ReplayCanvasOptions {
+  enableManualSnapshot?: boolean;
   quality: 'low' | 'medium' | 'high';
 }
 
-type GetCanvasManager = (options: CanvasManagerOptions) => CanvasManagerInterface;
+type GetCanvasManager = (options: CanvasManagerOptions) => void;
```

Comment:
I think this still needs to return `CanvasManagerInterface`, otherwise it will break for the non-manual case since rrweb depends on the return value of `getCanvasManager`